src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -232,6 +245,28 @@ public void assertMatchingGenotypesFromTileDB(File input, File expected, Locatab
         runGenotypeGVCFSAndAssertSomething(genomicsDBUri, expected, NO_EXTRA_ARGS, VariantContextTestUtils::assertVariantContextsHaveSameGenotypes, reference);
     }
 
+    @Test(dataProvider = "getGVCFsForGenomicsDBOverMultipleIntervals")
+    public void testGenotypeGVCFsMultiIntervalGDBPerformance(File input, File expected, List<Locatable> intervals, String reference) throws IOException {
+        final File tempGenomicsDB = GenomicsDBTestUtils.createTempGenomicsDB(input, intervals, true);
+        final String genomicsDBUri = GenomicsDBTestUtils.makeGenomicsDBUri(tempGenomicsDB);
+
+        final File output = createTempFile("genotypegvcf", ".vcf");
+
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.addReference(new File(reference))
+                .addArgument("V", genomicsDBUri)
+                .addOutput(output);
+        intervals.forEach(args::addInterval);
+        args.add("--" + GenomicsDBImport.MERGE_INPUT_INTERVALS_LONG_NAME);
+        args.add("--only-output-calls-starting-in-intervals");
+
+        Utils.resetRandomGenerator();
+        runCommandLine(args);
+
+        // if this isn't working it will take *FOREVER*
```

Comment:
I don't like this as a solution for our tests. This could easily cause inexplicable failures on travis that nobody knows how to solve. Furthermore using the test timeout functionality on testNG seems like a crude solution. I think you should expose the userIntervalList with a getter and then use that to explicitly assert that the intervals were merged correctly.

Reply:
I do need to test that the mergeIntervals arg works as expected. I added an actual comparison.  If you're still concerned about the runtime if something breaks, then I can reduce the number of intervals.